.prettierignore

```diff
@@ -0,0 +1 @@
+examples/specs/*.vl.json
```

Comment:
Can we put this rule in `.prettierrc.json`?

If we can, probably better to consolidate them in one place.

Reply:
I don't think it works yet. https://github.com/prettier/prettier/issues/3460